datumaro/plugins/pointcloud_format/format.py

```diff
@@ -0,0 +1,13 @@
+class PointCloudPath:
```

Comment:
There should be a license header with MIT license

Reply:
Resolved